code/modules/mob/living/carbon/human/human_helpers.dm

```diff
@@ -103,7 +103,9 @@
 
 
 /mob/living/carbon/human/IsAdvancedToolUser()
-	return 1//Humans can use guns and such
+	if(disabilities & MONKEYLIKE)
+		return FALSE
+	return TRUE//Humans can use guns and such

```

Comment:
return (disabilities & MONKEYLIKE)

Reply:
i think that'd be less reader-friendly, but if it's significant in performance i'll do that

Reply:
I'm not sure how faster it is, but if(X) return true, else return false just rubs me the wrong way.

Reply:
I'd prefer we not use the `return (disabilities & MONKEYLIKE)` style as it inhibits future expansion of the proc